ariadne/cardano/src/Ariadne/Wallet/Cardano/Kernel/DB/Util/IxSet.hs

```diff
@@ -157,6 +181,27 @@ member pk = isJust . view (Lens.at pk)
 size :: IxSet a -> Int
 size = IxSet.size . unwrapIxSet
 
+-- | Whether or not this 'IxSet' contains no elements.
+null :: IxSet a -> Bool
+null = IxSet.null . unwrapIxSet
```

Comment:
I'd rather define `null = Foldable.null` or `null = Container.null` or `Foldable.null = null`. Same for `toList`.

(I know it will increase diff with IOHK, but I think we should stop code imports after this one)

Reply:
I suppose we can just remove this function since it is defined in `Foldable` anyway.